### PR TITLE
Add Android download link

### PR DESCRIPTION
**Pull request by paulrouget**

### CODE REVIEW

**Comment by paulrouget** [comment]
I'm writing a blog post ([WIP](https://github.com/servo/blog.servo.org/pull/164)). Let's merge that once the blog post is ready to be published.

**Comment by atouchet** [comment]
Personally I would like to hold off on publicizing this until the crash issues have been figured out (ex. https://github.com/servo/servo/issues/21317). It seems clear that Servo has issues running on certain devices and I think it would be best to either solve that first or at least figure out what specs/Android OS version is required for the app to work properly so that information can be listed.

File: index.html [hunk 1]
```diff
@@ -50,10 +50,8 @@ <h3 class="description"><a href="https://servo.org/"><b>Servo</b></a>
                href="nightly/mac/servo-latest.dmg">macOS Build</a><br>
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/linux/servo-latest.tar.gz">Linux Build (64-bit)</a><br>
-<!-- Coming soon!
             <a class="btn btn-primary btn-large" role="button"
-               href="nightly/android/">Android Builds</a><br>
--->
+               href="nightly/android/servo-latest.apk">Android Builds</a><br>
```
Comment (atouchet) [inline index.html]: This should probably say "Build" rather than "Builds".

**Comment by paulrouget** [comment]
@atouchet Agreed. See https://github.com/servo/servo.org/issues/63

**Comment by paulrouget** [comment]
I think we should go ahead and add the button and publish [the blog post](https://github.com/servo/blog.servo.org/pull/164).

I know there are still crashers, and I have no idea how widespread it is, but the point of having testers is to get more data, and here, I would use more info about these crashers.

@jdm @larsbergstrom are you guys ok with that?

**Comment by larsbergstrom** [comment]
@paulrouget I made some comments in the blog post. Looks great overall and love seeing it coming together!

Also, if you'd like to get more crash info, would you be interested in possibly integrating https://sentry.io/for/android/ ? We have an enterprise license here at Mozilla and I'd be happy to work with the legal team to get the right permissions stuff in place so that we can get help users opt in to submitting crashes without having to file a bug with us every time.

**Comment by atouchet** [comment]
@paulrouget I'm fine with publishing if you think the crashes aren't widespread or want extra testers to help diagnose the problem. I just don't want to have another situation where new builds are pushed out before they are really ready. I feel that is what happened with the initial Windows nightly builds and that those could have used a few extra days of testing.

**Comment by paulrouget** [comment]
@larsbergstrom eventually, I'd love to get this crash reporting system in place (do we use that with Crow?)

@atouchet these builds are absolutely not "really ready" :) there are plenty of issues (functional, perf and rendering bugs). But we need help to tackle these down. This is not a product release.

**Comment by atouchet** [comment]
@paulrouget For sure. I meant "ready" more in the sense of the interface basically works and Servo can load a web page or two without crashing. I realize it's a long ways away from a real release.

**Comment by jdm** [comment]
Let's make the builds available and do the blog post later when we want to send more eyes towards the builds.

**Pull request merged**